.travis.yml

```diff
@@ -1,16 +1,25 @@
+sudo: required
+services:
+  - docker
+
 language: go
 go:
-  - 1.6
+  - 1.8
```

Comment:
Let's do `1.x` so we get `1.8.3` now and `1.9` soon.  I'd prefer to just always support the latest stable version of Go.